src/zks_wallet/wallet.rs

```diff
@@ -581,6 +671,47 @@ mod zks_signer_tests {
     }
 
     #[tokio::test]
+    async fn test_deposit() {
+        let private_key = "0x28a574ab2de8a00364d5dd4b07c4f2f574ef7fcc2a86a197f65abaec836d1959";
+        let amount: U256 = parse_units("0.01", "ether").unwrap().into();
+        println!("Amount: {}", amount);
+
+        let l1_provider = eth_provider();
+        let l2_provider = era_provider();
+        let wallet = LocalWallet::from_str(private_key)
+            .unwrap()
+            .with_chain_id(ERA_CHAIN_ID);
+        let zk_wallet =
+            ZKSWallet::new(wallet, Some(l2_provider.clone()), Some(l1_provider.clone())).unwrap();
+
+        let l1_balance_before = zk_wallet.eth_balance().await.unwrap();
+        let l2_balance_before = zk_wallet.era_balance().await.unwrap();
+        println!("L1 balance before: {}", l1_balance_before);
```

Comment:
we probably want to remove these before merging?

Reply:
There are many `println!` in different tests. I think that this is a nice feature because they are only shown when the test fails.